website/docs/sdk-reference/elixir.md

```diff
@@ -0,0 +1,235 @@
+| `name` | A unique identifier for this instance of `ConfigCat`. Defaults to `ConfigCat`.  Must be provided if you need to run more than one instance of `ConfigCat` in the same application. If you provide a `name`, you must then pass that name to all of the API functions using the `client` option. |
+
+
+## Anatomy of `get_value()`
+
+| Parameters      | Description                                                                                                  |
+| --------------- | ------------------------------------------------------------------------------------------------------------ |
+| `key`           | **REQUIRED.** Setting-specific key. Set on *ConfigCat Dashboard* for each setting.                           |
+| `default_value` | **REQUIRED.** This value will be returned in case of an error.                                               |
+| `user`          | Optional, *ConfigCat.User Object*. Essential when using Targeting. [Read more about Targeting.](advanced/targeting.md) |
+| `client`        | If you are running multiple instances of `ConfigCat`, provide the `client: :unique_name` option, specifying the name you configured for the instance you want to access. |
+```elixir
+value = ConfigCat.get_value(
+    "keyOfMySetting", # Setting Key
+    false, # Default value
+    ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92") # Optional User Object
+);
+```
+
+### User Object
+
+The [User Object](../advanced/user-object.md) is essential if you'd like to use ConfigCat's [Targeting](advanced/targeting.md) feature.
+```elixir
+user_object = ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92")
+```
+```elixir
+user_object = ConfigCat.User.new("john@example.com")
+```
+
+| Parameters   | Description                                                                                                                     |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `identifier` | **REQUIRED.** Unique identifier of a user in your application. Can be any value, even an email address.                         |
+| `email`      | Optional parameter for easier targeting rule definitions.                                                                       |
+| `country`    | Optional parameter for easier targeting rule definitions.                                                                       |
+| `custom`     | Optional `Map` for custom attributes of a user for advanced targeting rule definitions. e.g. User role, Subscription type. |
+```elixir
+user_object = ConfigCat.User.new("435170f4-8a8b-4b67-a723-505ac7cdea92", email: 'john@example', country: 'United Kingdom',
+                custom: ‰{SubscriptionType: 'Pro', UserRole: 'Admin'})
```

Comment:
I'm not sure about this, but this character seems a pretty weird to me. Maybe  ‰ -> % ?

Reply:
👀  fixed